### PR TITLE
Document iptables and container-selinux config for RancherD install on SELinux

### PR DESCRIPTION
**Pull request by catherineluse**
https://github.com/rancher/docs/issues/2916

### CODE REVIEW

**Review by rancher-max — COMMENTED** [review]
I wonder if for this documentation we should also include RHEL 8, since it's the enterprise version of centos? Or is just mentioning one what we generally do?

**Comment by catherineluse** [comment]
If the same configuration applies to RHEL 8 as well, I think it will be helpful to call that out. I'll update the PR.

File: content/rancher/v2.x/en/installation/requirements/_index.md [hunk 1]
```diff
@@ -148,3 +150,12 @@ Each node used should have a static IP configured, regardless of whether you are
 ### Port Requirements
 
 To operate properly, Rancher requires a number of ports to be open on Rancher nodes and on downstream Kubernetes cluster nodes. [Port Requirements]({{<baseurl>}}/rancher/v2.x/en/installation/requirements/ports) lists all the necessary ports for Rancher and Downstream Clusters for the different cluster types.
+
+# RancherD on SELinux Enforcing CentOS 8 or RHEL 8 Nodes
+
+Before installing Rancher on SELinux Enforcing CentOS 8 nodes, you must install `container-selinux` and `iptables`:
```
Comment (rancher-max) [inline content/rancher/v2.x/en/installation/requirements/_index.md:156]: We should call out rhel8 here too

File: content/rancher/v2.x/en/installation/install-rancher-on-linux/_index.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ RancherD must be launched on a Linux OS. At this time, only OSes that leverage s
 
 The Linux node needs to fulfill the [installation requirements]({{<baseurl>}}/rancher/v2.x/en/installation/requirements) for hardware and networking.
 
+To install RancherD on SELinux Enforcing CentOS 8 nodes, some [additional steps]({{<baseurl>}}/rancher/v2.x/en/installation/requirements/#rancherd-on-selinux-enforcing-centos-8-nodes) are required.
```
Comment (rancher-max) [inline content/rancher/v2.x/en/installation/install-rancher-on-linux/_index.md:41]: We should mention rhel 8 here too

**Review by rancher-max — CHANGES_REQUESTED** [review]

File: content/rancher/v2.x/en/installation/requirements/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/installation/requirements/_index.md:156]: updated

File: content/rancher/v2.x/en/installation/install-rancher-on-linux/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/installation/install-rancher-on-linux/_index.md:41]: updated

**Pull request merged**